src/heitech.zer0mqXt.core/Main/Zer0Mq.cs

```diff
@@ -0,0 +1,44 @@
+using heitech.zer0mqXt.core.Adapters;
+using heitech.zer0mqXt.core.infrastructure;
+
+namespace heitech.zer0mqXt.core.Main
+{
+    public class Zer0Mq : IZer0MqBuilder
```

Comment:
I'd have expected this class also to be suffixed with "Builder".
And then have another `static class Zer0Mq` which has a static method named `CreateDefaultBuilder`.
But I guess I am used to [Host.CreateDefaultBuilder](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.host.createdefaultbuilder).

Reply:
Hm I just wanted to have Zer0Mq.Go() in there because I like how that reads. For now I would leave it like this. We can discuss this of course, as adhering to conventions might be a good thing here.

On the other hand, defining our own unique hdk style might be nice for future branding purposes and such :D